framework/doc/content/source/outputs/Console.md

```diff
@@ -1,11 +1,16 @@
 <!-- MOOSE Documentation Stub: Remove this when content is added. -->
```

Comment:
```suggestion
```

Reply:
You should also remove "Outputs/Console" from the framework/doc/hidden.yml file.